### PR TITLE
Update customer-support.md

### PR DESCRIPTION
**Pull request by craigwillis-mm**
Added Sales Support section and other minor changes.

### CODE REVIEW

**Review by emilyhollinger — DISMISSED** [review]

File: operations/business-operations/customer-support.md [hunk 1]
```diff
@@ -103,3 +101,40 @@ The Support Team is driven by three key metrics:
 - Customer Satisfaction.  
 
 Details of these metrics for our Enterprise customers can be found [here](https://handbook.mattermost.com/operations/business-operations/analytics/metrics-definitions#support-tickets).
+
+## Sales Support
+
+Support is also provided for customers using our self-service portal which can be access [here](https://customers.mattermost.com/signup). Customer requesting assistance from here will be directed to [this](https://support.mattermost.com/hc/en-us/requests/new?ticket_form_id=360000852671) page.
```
Comment (amyblais) [inline operations/business-operations/customer-support.md]:
```suggestion
Support is also provided for customers using our self-service portal which can be accessed [here](https://customers.mattermost.com/signup). Customers requesting assistance from here will be directed to the [**Submit a Request**](https://support.mattermost.com/hc/en-us/requests/new?ticket_form_id=360000852671) page.
```

File: operations/business-operations/customer-support.md [hunk 2]
```diff
@@ -103,3 +101,40 @@ The Support Team is driven by three key metrics:
...
+
+## Ticket workflow (Sales Support)
+
+The Sales Support Team use a slightly difernet workflow to the Technical Support Team and this can be seen here:
```
Comment (amyblais) [inline operations/business-operations/customer-support.md]:
```suggestion
The Sales Support Team use a slightly different workflow to the Technical Support Team which is illustrated here:
```

File: operations/business-operations/customer-support.md [hunk 3]
```diff
@@ -103,3 +101,40 @@ The Support Team is driven by three key metrics:
...
+
+![](../../.gitbook/assets/Sales_Support_Process_Workflow.png)
```
Comment (amyblais) [inline operations/business-operations/customer-support.md:113]: This image doesn't show up. Has this been uploaded?

File: operations/business-operations/customer-support.md [hunk 4]
```diff
@@ -103,3 +101,40 @@ The Support Team is driven by three key metrics:
...
+
+The following information is requested when contacting Sales Support:
+
+|Field | Description |
+|------|-------------|
+| **CC**  |Add additional email addresses of those who you would like to receive notifications of this issue.|
+| **Subject** |A brief description of the issue.            |
+| **Category**     |Select a category that best fits the product area affected by the issue raised.            |
+| **Description**     |Provide as much detail as possible regarding the issue.              |
+| **Attachments**     |Add screenshots, logs, etc to the ticket here.
```
Comment (amyblais) [inline operations/business-operations/customer-support.md]:
```suggestion
| **Attachments**     |Add screenshots, logs, etc. to the ticket.    
```

File: operations/business-operations/customer-support.md [hunk 5]
```diff
@@ -103,3 +101,40 @@ The Support Team is driven by three key metrics:
...
+| **Description**     |Provide as much detail as possible regarding the issue.              |
+| **Attachments**     |Add screenshots, logs, etc to the ticket here.    
+
+### Sales Support categories
+
+The following categories are availble for submiting tickets to the Sales Support Team:
+
+|Name | Description |
+|------|-------------|
+| **Credit Card Issues**  |Problems regarding issues with credit card paymemnt|
```
Comment (amyblais) [inline operations/business-operations/customer-support.md]:
```suggestion
| **Credit Card Issues**  |Problems regarding issues with credit card payment.|
```

File: operations/business-operations/customer-support.md [hunk 6]
```diff
@@ -103,3 +101,40 @@ The Support Team is driven by three key metrics:
...
+| **General** |For issues that don't fit the other categories            |
```
Comment (amyblais) [inline operations/business-operations/customer-support.md]:
```suggestion
| **General** |For issues that don't fit the other categories.            |
```

File: operations/business-operations/customer-support.md [hunk 7]
```diff
@@ -103,3 +101,40 @@ The Support Team is driven by three key metrics:
...
+| **Licensing**     |For questions regarding the licensing of the Mattermost Product            |
```
Comment (amyblais) [inline operations/business-operations/customer-support.md]:
```suggestion
| **Licensing**     |For questions regarding the licensing of the Mattermost Product.            |
```

File: operations/business-operations/customer-support.md [hunk 8]
```diff
@@ -103,3 +101,40 @@ The Support Team is driven by three key metrics:
...
+| **Pricing**     |For questions regarding the pricing of Mattermost              |
```
Comment (amyblais) [inline operations/business-operations/customer-support.md]:
```suggestion
| **Pricing**     |For questions regarding the pricing of Mattermost.              |
```

File: operations/business-operations/customer-support.md [hunk 9]
```diff
@@ -103,3 +101,40 @@ The Support Team is driven by three key metrics:
...
+| **Product**     |For questions about the different products offered.  PLease note, for Technical Support please click [here](https://support.mattermost.com/hc/en-us/requests/new?ticket_form_id=360000640492)  |
```
Comment (amyblais) [inline operations/business-operations/customer-support.md]:
```suggestion
| **Product**     |For questions about the different products offered. For Technical Support please click [here](https://support.mattermost.com/hc/en-us/requests/new?ticket_form_id=360000640492).  |
```

File: operations/business-operations/customer-support.md [hunk 10]
```diff
@@ -103,3 +101,40 @@ The Support Team is driven by three key metrics:
...
+| **Renewals**     |For questions regarding license renewals  |
```
Comment (amyblais) [inline operations/business-operations/customer-support.md]:
```suggestion
| **Renewals**     |For questions regarding license renewals.  |
```

File: operations/business-operations/customer-support.md (hunk 3 above)
Comment (craigwillis-mm) [reply operations/business-operations/customer-support.md:113]: THat's odd, it was showing in preview mode for me yesterday, the file is here:

https://github.com/mattermost/mattermost-handbook/blob/0.2.1/.gitbook/assets/Sales_Support_Process_Workflow.png

**Review by amyblais — APPROVED** [review]

**Pull request merged**